### ISSUE TITLE
Search Input appears two times 

### ISSUE THREAD

**Issue by don-esteban** [apps-360/site-search-360-react#2]
Issue: See attached screenshot

Site developed with Gatsby/React

Steps to reproduce:

1. Visit [https://www.software-express.de/](https://www.software-express.de/)
2. Put some text in search input (upper right) and search
3. The result list contains the search input. That's correct.
4. Close the search result box
5. Visit another page
6. Perform a search again
7. The result list contains two search input fields now.

Thanks for Your support

<img width="921" alt="Bildschirmfoto 2022-04-01 um 15 07 50" src="https://user-images.githubusercontent.com/6874584/161280189-8928d643-77f4-4dc7-bad8-80360a6389a9.png">

**Comment by JVankat** [comment]
To me this looks like an integration issue rather than an issue with the component.

You're not using the search inputs rendered by this plugin, is that correct?

The issue seems to be the following:

1. the search inputs don't get recreated on navigation
2. the SiteSearch360 component gets recreated after you navigate to another page --> the componentDidMount hook is called and it rebinds all event listeners on the inputs
3. nevertheless, the events have already been bound, so duplicate events get created on the search inputs which results in this behavior

So, the solution should be either making sure the SiteSearch360 component only mounts when the search inputs are recreated (probably integrating our component directly in the header), or [integrating the search script manually](https://docs.sitesearch360.com/installation?ss360SearchTerm=Advanced%20Configuration).

**Comment by JVankat** [comment]
Yes, that should be possible, we're using the vue app in our control panel multiple times on a page (with different configurations), and the vue app is basically the same as the react app.

So, as long as you make sure the events aren't bound multiple times, everything should work as expected.
